vtex/toolbelt
# Remove special render build

#### What is the purpose of this pull request?
Removes all special build steps from the `render` service on `watch` and `publish`. Now every folder in the app is uploaded normally, with every file as-is.

Note that `eslint` was lost in the process. We need to think of a way to allow services to configure their own checks (via `manifest.json` configuration, maybe?).

#### What problem is this solving?
The fact that we are changing our approach in how the render apps in a workspace work makes any build step unnecessary. Instead of individual builds, all apps in a workspace will be compiled by a single webpack process. Adieu, SystemJS.

#### How should this be manually tested?
Thoroughly. 😝 
But really just notice that there's no more `.build/` folder upon watching and publishing.

#### Screenshots or example usage
```
guilherme@trinsic ~/P/V/hello-world (master)> releasy patch stable
Release: patch
cat: no such file or directory: package.json
package.json not found, switching to manifest.json
Old version: 0.1.18
New version: 0.1.19
prompt: Are you sure?:  (yes)
Starting release...
Version bumped to 0.1.19
File manifest.json added > git add manifest.json
File manifest.json committed > git commit manifest.json -m "Release v0.1.19"
Tag created: v0.1.19 > git tag v0.1.19 -m "Release v0.1.19"
Pushed commit and tags > git push && git push --tags
info:    Published app vtex.hello-world@0.1.19 successfully
Post releasy > vtex publish
All steps finished successfuly.
```

`curl "http://api.beta.vtex.com/pokestore/render-demo-aug-16/apps/vtex.hello-world@0.1.19/files"`

```
{
    "files": [
        {
            "path": "manifest.json",
            "hash": "\"F4B6C28F22883EB395F8A08C93D800AE\""
        },
        {
            "path": "render/index.js",
            "hash": "\"94195661B62E935900F2DA145681A710\""
        },
        {
            "path": "render/nested.js",
            "hash": "\"74E526A44E3AADAEDB17412216D2F3D8\""
        },
        {
            "path": "render/routes.json",
            "hash": "\"66A651DC7147B1094F4412A59EA3082D\""
        },
        {
            "path": "render/simple.js",
            "hash": "\"5195EF02CEE80BDCCFA2C7C7B46BB011\""
        }
    ],
    "nextMarker": null
}
```

#### Types of changes
